gui/docks/falsecolordock.cpp

```diff
@@ -311,3 +312,12 @@ void FalseColorDock::screenshot()
 	io.setFileCategory("Screenshot");
 	io.writeImage(output);
 }
+
+bool FalseColorDock::eventFilter(QObject *obj, QEvent *event)
```

Comment:
What is the purpose of this? Does the leve event in scaledview not suffice? (Do we do the same for banddock?)